simapp/README.md

```diff
@@ -0,0 +1,41 @@
+---
+order: false
+---
+
+# simapp
+
+simapp is an application built using the Cosmos SDK for testing and educational purposes. 
+
+## Running testnets with `simd` 
+
+If you want to spin up a quick testnet with your friends, you can follow these steps. 
+Unless otherwise noted, every step must be done by everyone who wants to participate
+in this testnet.
+
+1. `$ make build`. This will build the `simd` binary and install it in your Cosmos SDK repo, 
+    inside a new `build` directory. The following instructions are run from inside 
+    that directory. 
+2. If you've run `simd` before, you may need to reset your database before starting a new
+    testnet: `$ ./simd unsafe-reset-all` 
+3. `$ ./simd init [moniker]`. This will initialize a new working directory, by default at 
+    `~/.simapp`. You need a provide a "moniker," but it doesn't matter what it is.
+4. `$ ./simd keys add [key_name]`. This will create a new key, with a name of your choosing. 
+    Save the output of this command somewhere; you'll need the address generated here later.
+5. `$ ./simd add-genesis-account  $(simd keys show [key_name] -a) [amount]`, where `key_name`
+    is the same key name as before; and `amount` is something like `10000000000000000000000000stake`.
+6. `$ ./simd gentx [key_name] [amount] --chain-id [chain-id]`. This will create the
+    genesis transaction for your new chain. 
+7. Now, one person needs to create the genesis file `genesis.json` using the genesis transactions 
+   from every participant. TODO: ask Marko for details on how he sets this up 
```

Comment:
@marbar3778 can you add notes on how you generate the shared `genesis.json` files?